commit 2b4c45dc7d1b149dd4742d0579cdba044bde3482
Author: Oren Givon <oren.givon@intel.com>
Date:   Thu Aug 31 13:15:09 2017 +0300

    iwlwifi: add a new a000 device
    
    commit d048b36b9654c4e0cf0d3576be2d1ed2a3084c6f upstream.
    
    Add a new a000 device with PCI ID (0x2720, 0x0030).
    
    Signed-off-by: Oren Givon <oren.givon@intel.com>
    Signed-off-by: Luca Coelho <luciano.coelho@intel.com>
    Cc: Thomas Backlund <tmb@mageia.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/wireless/intel/iwlwifi/pcie/drv.c b/drivers/net/wireless/intel/iwlwifi/pcie/drv.c
index 3416c6155996..e0966f656376 100644
--- a/drivers/net/wireless/intel/iwlwifi/pcie/drv.c
+++ b/drivers/net/wireless/intel/iwlwifi/pcie/drv.c
@@ -577,6 +577,7 @@ static const struct pci_device_id iwl_hw_card_ids[] = {
 	{IWL_PCI_DEVICE(0x34F0, 0x0070, iwla000_2ax_cfg_hr)},
 	{IWL_PCI_DEVICE(0x2720, 0x0078, iwla000_2ax_cfg_hr)},
 	{IWL_PCI_DEVICE(0x2720, 0x0070, iwla000_2ac_cfg_hr_cdb)},
+	{IWL_PCI_DEVICE(0x2720, 0x0030, iwla000_2ac_cfg_hr_cdb)},
 	{IWL_PCI_DEVICE(0x2720, 0x1080, iwla000_2ax_cfg_hr)},
 #endif /* CONFIG_IWLMVM */
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1181_2b4c45dc7d1b149dd4742d0579cdba044bde3482.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1181_2b4c45dc7d1b149dd4742d0579cdba044bde3482.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
